### PR TITLE
update the error code for get chef-client node and list

### PR DESCRIPTION
**Pull request by hmunfru**
#### Reviewers
@jframos @jesuspg

#### Description
https://jira.tid.es/browse/CLAUDIA-4210
https://jira.tid.es/browse/CLAUDIA-4209

#### Test
Tested locally agains experimentacion testbed

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/sdc/dao/impl/ChefClientDaoRestImpl.java [hunk 1]
```diff
@@ -140,9 +140,9 @@ public void deleteChefClient(String chefClientName, String token) throws CanNotC
 
     }
 
-    /*
-     * (non-Javadoc)
-     * @see com.telefonica.euro_iaas.sdc.dao.ChefClientDao#getChefClient(java.lang.String)
+    /**
+     *
+     *
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/sdc/dao/impl/ChefClientDaoRestImpl.java]: Come on!!!

File: core/src/main/java/com/telefonica/euro_iaas/sdc/manager/impl/NodeManagerImpl.java [hunk 2]
```diff
@@ -215,11 +214,9 @@ public ChefClient chefClientload(String chefClientName, String token) throws Che
         try {
             chefClient = chefClientDao.getChefClient(chefClientName, token);
         } catch (EntityNotFoundException e) {
-            // String message =
-            // " An error ocurred invoing the Chef server to load ChefClient named "
-            // + chefClientName;
-            // log.info(message);
-            throw e;
+            String men = "The node is not in the chef-server: " + chefClientName + " : " + e.getMessage();
+            log.warn ("The node is not in the chef-server.");
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/sdc/manager/impl/NodeManagerImpl.java]: Why do you not do log.warn( men )?

File: core/src/main/java/com/telefonica/euro_iaas/sdc/manager/impl/NodeManagerImpl.java [hunk 3]
```diff
@@ -166,7 +165,7 @@ private void chefClientDelete(String vdc, String chefClientName, String token) t
             // eliminacion del chefClient
             chefClientDao.deleteChefClient(chefClientName, token);
 
-        } catch (CanNotCallChefException e) {
+        }  catch (CanNotCallChefException e) {
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/sdc/manager/impl/NodeManagerImpl.java]: Do you have here 2 spaces before catch function?

**Comment by flopezag** [comment]
LGFM

**Comment by jframos** [comment]
LGTM

**Pull request merged**